components/launcher/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "habitat-launcher"
+version = "0.0.0"
+authors = ["Jamie Winsor <reset@habitat.sh>"]
+workspace = "../../"
+
+[[bin]]
+name = "hab-launch"
+doc = false
+
+[dependencies]
+ansi_term = "*"
+env_logger = "*"
+# JW TODO: core has external deps that we don't want, libarchive/libsodium. We should either
```

Comment:
Was there supposed to be a choice of things to do here?  Either put these things behind a feature flag or ... something else?